recipe/meta.yaml

```
@@ -83,7 +83,7 @@ outputs:
         # above, we need most of these at build time, so they are mostly
         # host-like
         - apr {{ apr }}.*
-        - boost {{ boost }}.*
+        - libboost-python-devel {{ boost }}.*
```

Version constraint will be handled by run-export from host
```suggestion
        - libboost-python-devel
```